<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Network</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SIP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Skype</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VOEP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Advance</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page_4">
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>57</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Genral</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>57</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Network</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>SIP options</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Use TLS:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>SIP Protocol:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_4">
             <item>
              <property name="text">
               <string>UDP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TCP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TLS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>IPV6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UDP6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TCP6</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Codecs:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox_5">
             <property name="toolTip">
              <string>语音编码</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>NAT Detect:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox_6">
             <item>
              <property name="text">
               <string>NONE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>STUN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>TURN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ICE</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>    STUN Server:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>
Advanced</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Skype Bridge Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="toolTip">
              <string>Using your skype as network transport layer
This method supply raliable transport when call initialized, 
but may be little problem when call initializing ,
because we must check your skype installation, 
run state , login state and app to skype connect status.</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>79</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QListView" name="listView">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>100</y>
           <width>256</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>57</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Skype</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_5">
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>80</y>
           <width>57</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>VOEP</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_6">
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>57</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Advanced</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
